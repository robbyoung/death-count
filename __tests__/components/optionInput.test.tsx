import React from 'react';
import renderer from 'react-test-renderer';
import OptionInput from '../../app/components/optionInput';

describe('Option Input Component', () => {
    it('renders correctly', () => {
        const component = renderer.create(<OptionInput
            onSubmit={() => undefined}
            placeholder='placeholder'
        />);
        expect(component.toJSON()).toMatchSnapshot();
    });
});