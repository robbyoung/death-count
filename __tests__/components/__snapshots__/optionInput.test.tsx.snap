// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Option Input Component renders correctly 1`] = `
<View
  style={
    Object {
      "alignContent": "center",
      "borderRadius": 5,
      "flexDirection": "row",
      "justifyContent": "center",
      "padding": 5,
      "width": "100%",
    }
  }
>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    placeholder="placeholder"
    rejectResponderTermination={true}
    style={
      Object {
        "backgroundColor": "#ffffff",
        "borderBottomLeftRadius": 10,
        "borderTopLeftRadius": 10,
        "flexGrow": 100,
        "fontSize": 22,
        "height": 50,
        "paddingLeft": 10,
      }
    }
    underlineColorAndroid="transparent"
    value=""
  />
  <View
    accessible={true}
    clickable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "backgroundColor": "transparent",
          },
          Object {
            "color": undefined,
          },
          Object {
            "backgroundColor": "#e84641",
            "borderBottomRightRadius": 10,
            "borderTopRightRadius": 10,
            "color": "#ffffff",
            "fontSize": 25,
            "height": 50,
            "textAlign": "center",
            "textAlignVertical": "center",
            "width": 50,
          },
          Object {
            "fontFamily": "FontAwesome5FreeSolid",
          },
        ]
      }
    >
      ï€Œ
    </Text>
  </View>
</View>
`;
